components/Card/Title.d.ts

```diff
@@ -0,0 +1,10 @@
+import React = require('react');
```

Comment:
O que acha de importar assim:

```suggestion
import React from 'react';
```